priv/repo/seeds/accounts.exs

```diff
@@ -245,13 +245,20 @@ defmodule Bokken.Repo.Seeds.Accounts do
 
     photo = nil
 
+    codemonkey = Enum.random(["Ze","Antonio","Mario","Julio","Durate","Dinis"])
+    lightbot = Enum.random(["Ze","Antonio","Mario","Julio","Durate","Dinis"])
+    codewars = Enum.random(["Ze","Antonio","Mario","Julio","Durate","Dinis"])
+
     ninja =
       Enum.into(names, %{
         user_id: user_id,
         guardian_id: guardian_id,
         photo: photo,
         belt: belt,
-        birthday: birthday
+        birthday: birthday,
+        codemonkey: codemonkey,
+        lightbot: lightbot,
+        codewars: codewars
```

Comment:
A ninja `embeds_many` socials and a `Social` is just an embedded struct.
For example, a `Social` could be:
```elixir
%{"socials" => %{"codemonkey" => %{"name" => "codemonkey", "username" => "Rui"}}}
```
So that, a ninja with those three socials would have something like:
```elixir
%{
  "socials" => %{
    "codemonkey" => %{"name" => "codemonkey", "username" => "Rui"},
    "lightbot" => %{"name" => "lightbot", "username" => "Rui"},
    "codewars" => %{"name" => "codewars", "username" => "Rui"}
  }
}
```
It's like we are literally storing JSON on a database.

Reply:
I'm not getting how should I implement this suggestion :sweat_smile: